internal/charmstore/export_test.go

```diff
@@ -3,4 +3,7 @@
 
 package charmstore
 
-var TimeToStamp = timeToStamp
+var (
+	TimeToStamp          = timeToStamp
+	MustParseResolvedURL = mustParseResolvedURL
```

Comment:
Given that this is only defined in test code, I'd suggest just exporting it there rather than adding it to export_test.go, which is really only for things in production code that we don't want to export to the public API.